grazel-gradle-plugin/src/main/kotlin/com/grab/grazel/gradle/dependencies/Dependencies.kt

```diff
@@ -55,7 +46,7 @@ internal val DEP_GROUP_EMBEDDED_BY_RULES = listOf(
     "org.jetbrains.kotlin"
 )
 
-data class ExcludeRule(
```

Comment:
Will be removed in upcoming PR.

Reply:
Maybe better to add Deprecated annotation